specification/cognitiveservices/OpenAI.Inference/models/whisper.common.tsp

```diff
@@ -0,0 +1,57 @@
+import "@typespec/rest";
+import "@typespec/http";
+
+import "./transcription.create.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+
+namespace Azure.OpenAI;
+
+@doc("Defines the format of the output.")
+enum AudioTranscriptionFormat {
+    @doc("JSON format. The translation/transcription response will only contain 'text'.")
+    json: "json",
+
+    @doc("Text format. The translation/transcription response will be of type text/plain.")
+    text: "text",
+
+    @doc("SRT format. The translation/transcription response will be of type text/plain.")
+    srt: "srt",
+
+    @doc("Verbose JSON format.The translation/transcription response will contain additional information.")
+    verbose_json: "verbose_json",
+
+    @doc("VTT format. The translation/transcription response will be of type text/plain.")
+    vtt: "vtt",
+}
+
+@doc("Transcription segment.")
+model AudioTranscriptionSegment {
```

Comment:
Raised this in Ralf's PR, too, but I think we're missing two fields: `tokens`, an array of `int32`s, and `seek`, another `int32`

Reply:
Added 👍

Reply:
These were added in the json so I will resolve this thread.